src/backend/distributed/operations/shard_split.c

```diff
@@ -742,7 +726,7 @@ CreateAndCopySplitShardsForShardGroup(HTAB *mapOfShardToPlacementCreatedByWorkfl
 static void
 DoSplitCopy(WorkerNode *sourceShardNode, List *sourceColocatedShardIntervalList,
 			List *shardGroupSplitIntervalListList, List *destinationWorkerNodesList,
-			char *snapShotName)
+			char *snapShotName, DistributionColumnMap *distributionColumnOverrides)
```

Comment:
Do we need this map to be passed around? I see there is only one Column name to be used as of now.

Reply:
In case of partitioned tables there is an entry for every partition. Partitions could perhaps be handled separately, but in general we might want to introduce a function that can handle multiple co-located tables, so this seems like the right infrastructure.